### DISCUSSION TITLE
AUTH_REDIRECT_PROXY_URL limitations

### DISCUSSION THREAD

**Discussion by lightcap** [nextauthjs/next-auth#8575] [Help]
The [docs](https://authjs.dev/guides/basics/deployment#securing-a-preview-deployment) say that `AUTH_REDIRECT_PROXY_URL` can be used to proxy preview deployments. But this doesn't seem to be true, at least not wholesale.

The only place I see in the code that [uses that environment variable is for Github](https://github.com/nextauthjs/next-auth/blob/b1c46809f5f623f4f7bb9f42fbab8d7866825997/apps/dev/nextjs/pages/api/auth/%5B...nextauth%5D.ts#L111). 

I've tried using `AUTH_REDIRECT_PROXY_URL` with Cognito with no luck. I'm either missing something or the docs should be updated to note that it only works with Github.

Thoughts?

**Comment by samtgarson** [comment]
It seems like that env var is being used as a default for [this value](https://github.com/nextauthjs/next-auth/blob/b5de00016f37047b3b84700757556f71c982f76d/packages/next-auth/src/lib/env.ts#L15), which is then used in a few other places. 

I'm not sure why the Github provider specifically references that env var...

**Comment by drewdelano** [comment]
@balazsorban44 you wrote this didn't you? can you shed some light here?
